icons/infinity.svg

```diff
@@ -9,5 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M18.178 8c5.096 0 5.096 8 0 8-5.095 0-7.133-8-12.739-8-4.585 0-4.585 8 0 8 5.606 0 7.644-8 12.74-8z" />
+  <path d="M15.3,15c0.7,0.6,1.6,1,2.7,1c2.2,0,4-1.8,4-4s-1.8-4-4-4c-1,0-2,0.4-2.7,1l-6.7,6C8,15.6,7,16,6,16 c-2.2,0-4-1.8-4-4s1.8-4,4-4c1,0,2,0.4,2.7,1L15.3,15z" />
```

Comment:
I can definitely see this moving forward, but checking with a curvature tool, the lines are somewhat wonky:
![image](https://user-images.githubusercontent.com/17746067/228440612-620fc4ae-e311-4732-b7c9-eed3c9ddf97e.png)

Here's what a perfect quadratic curve would look like (adjusted for rounding via optimization of course):
```suggestion
  <path d="M12 12c-2-2.67-4-4-6-4a4 4 0 1 0 0 8c2 0 4-1.33 6-4Zm0 0c2 2.67 4 4 6 4a4 4 0 0 0 0-8c-2 0-4 1.33-6 4Z" />
```

Reply:
> checking with a curvature tool

What are you using for this?

Reply:
https://www.figma.com/community/plugin/932729267119671976/Curvature